src/components/Activity/TransactionProgress.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ProgressBar, GU } from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { ActivityStatusType } from './prop-types'
+import { norm } from '../../lib/math-utils'
+import useNow from '../../hooks/useNow'
+import { ACTIVITY_STATUS_CONFIRMED, ACTIVITY_STATUS_PENDING } from './symbols'
```

Comment:
Should we delete `./symbols` and  import these from the `Activity` provider instead (or import symbols from the provider) ? Otherwise i think the progress bar would never show 😄

Reply:
Yes! I removed them from the `Activity` provider instead :-)